examples/google-cloud-speech/output/google-cloud-speech/acceptance/google/cloud/speech/v1/speech_smoke_test.rb

```diff
@@ -22,10 +21,9 @@
 
 describe "SpeechSmokeTest v1" do
   it "runs one smoke test with recognize" do
-
     speech_client = Google::Cloud::Speech.new(version: :v1)
     language_code = "en-US"
-    sample_rate_hertz = 44100
+    sample_rate_hertz = 44_100
```

Comment:
I forgot about this rule. I dislike this. Thoughts on excluding this change?

Reply:
Sure, I'll revert to `44100`.